pkg/controller/jenkins/configuration/user/seedjobs/seedjobs.go

```diff
@@ -164,6 +164,14 @@ func (s *SeedJobs) EnsureSeedJobs(jenkins *v1alpha2.Jenkins) (done bool, err err
 		if err != nil {
 			return false, err
 		}
+
```

Comment:
the complexity of this method is way too high already and adding more ifs only makes things worse, there must be a better way